.github/config.yml

```diff
@@ -4,3 +4,35 @@
 # Can be either a string :
 requestInfoReplyComment: >
   We would appreciate it if you could provide us with more info about this issue/pr!
```

Comment:
We would appreciate it if you could provide us with more info about this issue/prsssssssss!